src/components/experiment/ExperimentTable.tsx

```diff
@@ -100,6 +56,48 @@ function ExperimentTable() {
     []
   );
 
+  const { timezone, toFormattedDateTime } = useFormattedDateTime({
+    shouldUseTimeZone: true,
+  });
+
+  const columns = [
+    {
+      title: 'Proposal ID',
+      field: 'proposal.proposalId',
+    },
+    {
+      title: 'Principal investigator',
+      render: (rowData: RowType) => getFullUserName(rowData.proposal.proposer),
+    },
+    {
+      title: 'Proposal',
+      field: 'proposal.title',
+    },
+    {
+      title: `Experiment start (${timezone})`,
+      field: 'startsAt',
+      render: (rowData: RowType) => toFormattedDateTime(rowData.startsAt),
+    },
+    {
+      title: `Experiment end (${timezone})`,
+      field: 'endsAt',
+      render: (rowData: RowType) => toFormattedDateTime(rowData.endsAt),
```

Comment:
This will probably result in a console warning from material-table because that is how it works for some reason. That's why I suggested keeping the columns outside the component but just maping here inside and adding the new fields that are formatted dates. Something like it is done here: https://github.com/UserOfficeProject/user-office-frontend/blob/develop/src/components/call/CallsTable.tsx#L218